File: si.21.07.gde_je_moj_ljubimac/Aplikacija/Domain/Korisnik.cs
using System;
using System.Collections.Generic;
using Microsoft.AspNetCore.Identity;

namespace Domain
{
    public class Korisnik : IdentityUser
    {
        public string DisplayName { get; set; }
        public string Adresa { get; set; }
        public int? Ocena { get; set; }
        public int Status_naloga { get; set; }
        public int Tip_korisnika { get; set; }
        public List<Oglas> Oglasi { get; set; } = new List<Oglas>();
        public List<Ocena> Ocene { get; set; } = new List<Ocena>();
    }
}

File: si.21.07.gde_je_moj_ljubimac/Aplikacija/client/src/app/models/korisnik.ts
import { IOglas } from "./oglas";

export interface IKorisnikEnvelope {
    korisnici: IKorisnik[];
    korisnikBroj: number;
}

export interface IKorisnik{

    id : string;
    displayName: string;
    adresa : string; 
    ocena : number;
    statusNaloga : number;
    tipKorisnika : number;
    email: string;
    username: string;
    oglasi: IOglas[];
}

export interface IOcena{
    ocenaValue: number;
    korisnikId:string;
}
